rpc_jobs/rpc_octavia_gating.yml

```diff
@@ -0,0 +1,46 @@
+ - project:
+    name: "rpc-octavia"
```

Comment:
nit: this should probably be renamed, for consistency, RPC-octavia-Jobs

The filename itself should probably be rpc-octavia.yml also for consistency.

Reply:
@evrardjp That is the old standard. The name here is correct.

Reply:
OK.